resemble.js

```diff
@@ -797,7 +783,7 @@ URL: https://github.com/Huddle/Resemble.js
 			},
 			compareTo: function(secondFileData){
 				return getCompareApi(secondFileData);
-			},
+			}
```

Comment:
Just butting in here, but this comma shouldn't be removed.
```suggestion
			},
```